news/2017-08-30-qat-gazette-august-2017.md

```diff
@@ -0,0 +1,41 @@
+# QAT Gazette - August 2017
```

Comment:
needs a date/post header at the top a la:

```
 ---
 layout: post
 title: "Scorewatch: August Week 3"
 date: 2017-08-31 09:00:00 +0000
 ---
```

substituting values where appropriate, +0000 is UTC time. date yours (and this article) for the 3rd of september, this will mean you need to rename the article filename to (2017-09-03-qat-gazette etc) as well

Reply:
Heading added, penned for 3rd September, assuming I can leave time as +0000